component/service/src/test/java/org/exoplatform/social/rest/impl/users/UserRestResourcesTest.java

```diff
@@ -535,10 +535,6 @@ public void testGetUserAvatarForAnonymous() throws Exception {
     response = service("GET", getURLResource("users/" + user + "/avatar"), "", null, null);
     assertNotNull(response);
     assertEquals(404, response.getStatus());
-
-    response = service("GET", getURLResource("users/" + LinkProvider.DEFAULT_IMAGE_REMOTE_ID + "/avatar"), "", null, null);
-    assertNotNull(response);
-    assertEquals(200, response.getStatus());
```

Comment:
Should remain true and we shouldn't modify this.

Reply:
fixed